Kbs.Wpf/Damage/Read/Details/ReadDamageDetailsPage.xaml.cs

```diff
@@ -33,6 +33,15 @@ public ReadDamageDetailsPage(int damageId, INavigationManager navigationManager)
         ViewModel.Date = damage.Date;
         ViewModel.Image = damage.Image.ToImageSource();
         ViewModel.BoatName = boat.Name;
+
+        if (damage.Status == Business.Damage.DamageStatus.Solved)
+        {
+            ViewModel.SolveButtonEnabled = false;
+        } else
+        {
+            ViewModel.SolveButtonEnabled = true;
+        }
```

Comment:
(optioneel) Kan je in een expression statement doen:
```suggestion
        ViewModel.SolveButtonEnabled = damage.Status != Business.Damage.DamageStatus.Solved;
```

Reply:
Kan, maar dan word ik confused